dags/copy_deduplicate.py

```diff
@@ -87,6 +87,30 @@
         resources=resources,
     )
 
+    with TaskGroup('copy_deduplicate_all_downstream_external') as copy_deduplicate_all_downstream_external:
```

Comment:
I like that the `_downstream_external` suffix is precise/unambiguous, but it is a bit lengthy, and in the default tree view that will push the task-specific part of the task ID further behind the run results grid where it'll get more hidden/cut-off.

Maybe just `_downstream` _or_ `_external` would still be clear enough, while being more succinct?

Reply:
`_external` is probably good enough.